Crawl/CognitiveServiceEntityLinking/CognitiveServiceEntityLinking.cs

```diff
@@ -39,27 +39,35 @@ public static Task<HttpResponseMessage> Run(HttpRequestMessage req, TraceWriter
                     if (textBuilder.Length == 0)
                         return null;
 
-                    return Services.Limit(textBuilder.ToString(), 10240);
+                    string text = Services.Limit(textBuilder.ToString(), 10240);
```

Comment:
>10240 [](start = 73, length = 5)

Why 10k here but 5k in CreateRequestFromText()?

Reply:
Per Markus, this is due to ANSI vs UTF-16 character encoding (10k vs 5k per 10k byte buffer). Will add comment here, to make it clearer.